drivers/SymfonyValetDriver.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class SymfonyValetDriver extends ValetDriver
+{
+    /**
+     * Determine if the driver serves the request.
+     *
+     * @param  string  $sitePath
+     * @param  string  $siteName
+     * @param  string  $uri
+     * @return bool
+     */
+    public function serves($sitePath, $siteName, $uri)
+    {
+        return file_exists($sitePath.'/web/app_dev.php') &&
+               (file_exists($sitePath.'/bin/symfony_requirements') ||
+                file_exists($sitePath.'/app/SymfonyRequirements.php'));
```

Comment:
Maybe we could rename this to "SymfonyStandardEditionValetDriver" to be more specific. I don't think it makes sense making anything more generic than this.